sources/packages/backend/apps/api/src/utilities/class-transform/to-boolean.ts

```diff
@@ -0,0 +1,40 @@
+import { Transform } from "class-transformer";
+
+/**
+ * Converts a plain property to boolean.
+ * @returns converted boolean value.
+ */
+export const ToBoolean = () => {
```

Comment:
For my knowledge: should we add return type for these functions?

Reply:
We can, just the return type will be a long repetition of the next line. If there is an understand that it would be clear in this way I can add it as shown below:
```ts
/**
 * Converts a plain property to boolean.
 * @returns converted boolean value.
 */
export const ToBoolean = (): ((target: any, key: string) => void) => {
  return (target: any, key: string) => {
    return Transform(
      ({ obj }) => {
        return valueToBoolean(obj[key]);
      },
      {
        toClassOnly: true,
      },
    )(target, key);
  };
};
```